docs/source/queries/values.md

```diff
@@ -30,6 +31,22 @@ session
     .query("INSERT INTO ks.tab (a, b) VALUES(?, ?)", (2_i32, "Some text"))
     .await?;
 
+// Sending an integer and a string using a named struct
```

Comment:
I feel that we should explicitly say that names of fields don't matter, only order. Otherwise someone might think that the driver is smart enough to match struct fields with the columns.

Maybe we could change `IntString` to have fields like:
```rust
struct IntString {
    first_column: i32,
    second_column: String,
}
```
and add some comment about it.

Reply:
I don't feel like it's an issue, but I'm biased as a long time Scylla developer, so your opinion is more important in this matter :) I'll push an updated version in a while

Reply:
done